src/test_workflow/test_recorder/test_recorder.py

```diff
@@ -33,6 +34,14 @@ def __init__(self, test_run_id: str, test_type: str, tests_dir: str) -> None:
         self.local_cluster_logs = LocalClusterLogs(self)
         self.remote_cluster_logs = RemoteClusterLogs(self)
         self.test_results_logs = TestResultsLogs(self)
+        self.base_path = base_path
+
+    def _get_file_path(self, base_path: str, component_name: str, component_test_config: str) -> str:
+        if base_path.startswith("https"):
```

Comment:
Do `https://` to be safe. I can have a folder called `https`.

Reply:
Make sense for me. Just updated. Thanks.